### DISCUSSION TITLE
It is not possible to delegate using ledger

### DISCUSSION THREAD

**Discussion by ivvi1993** [jeluard/swipealot#576] [General]
<img width="554" alt="image" src="https://github.com/paritytech/governance-ui/assets/121741268/f292ec4c-78be-4a89-938d-fe1243ea1c3a">

It is not possible to delegate using ledger, I've tried many times and the error keeps repeating; someone know how fix ir? "Call vector exceeds limit"

**Comment by jeluard** [comment]
@ivvi1993 ledger doesn't currently support the required functions. I will keep you updated.

**Reply by rossbulat** [reply]
Can we reference a link here to where this is being discussed - I assume on Zondax Polkadot issues?

**Comment by rossbulat** [comment]
I imagine this will ultimately be resolved when the upcoming connect component is ready to be used between our apps. I think this should be the focus here UI-wise.
